lib/ansible/modules/system/filesystem.py

```diff
@@ -135,6 +144,17 @@ def create(self, opts, dev):
             cmd = "%s %s %s '%s'" % (mkfs, self.MKFS_FORCE_FLAGS, opts, dev)
         self.module.run_command(cmd, check_rc=True)
 
+    def get_uuid(self, dev):
+        """ Fetch the UUID """
+        blkid = self.module.get_bin_path('blkid', required=True)
```

Comment:
Reuse the result of [previous `get_bin_path` call](https://github.com/ansible/ansible/blob/8534178b1d28f2134da8f6b6634879157b5e34e9/lib/ansible/modules/system/filesystem.py#L397).